installation/azure/vmclarity.bicep

```diff
@@ -0,0 +1,151 @@
+targetScope = 'subscription'
+
+@description('Location for the VMClarity resource group')
+param location string = 'eastus'
+
+@description('Username for the Virtual Machine.')
+param adminUsername string
+
+@description('SSH Key for the Virtual Machine. SSH key is recommended.')
```

Comment:
Maybe worth mentioning this is the public key

Reply:
updated the description